### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by johasalinas**
This PR has multiple improvements:
- The graphics now has a 100% width with a min-width of 700px (small screens) and a max-width of 1200px (Desktop screens).
- The graphics container has been centered.
- The controls are full clickable that means you can click the square instead of having to click the specific checkbox.

This is the link of the issue: UI gets convoluted on a narrow screen https://github.com/intermine/bluegenes-tissue-expression-visualizer/issues/6

Note: The min-width of 700px was added because below that is not possible to avoid the overlap labels, this is an issue with the chart.js library itself.

Note: Looks like the tests are broken from before. This issue has been created to track: https://github.com/intermine/bluegenes-tissue-expression-visualizer/issues/10

image
image

### CODE REVIEW

File: src/style.less [hunk 1]
```diff
@@ -1,3 +1,11 @@
+.main-container {
```
Comment (heralden) [inline src/style.less]: Would it work to have these styles apply to `.rootContainer` instead? The containing div element won't have the `.main-container` class when used in Bluegenes.

**Review by heralden — CHANGES_REQUESTED** [review]
Please check my comment before I test this further. [=

**Comment by johasalinas** [comment]
@uosl Ready to update .rootContainer with .main-container styles. and works

In this PR you also have the changes after changing url from https to http trying to solve: [FAIL tests / tissueExpression.test.js: Cannot find module 'es6-promise' from 'promise.js' # 10](https://github.com/intermine/bluegenes-tissue-expression-visualizer/issues/10)

**Review by heralden — APPROVED** [review]
Thank you, this works great. :100:

**Pull request merged**